conformance/utils/kubernetes/helpers.go

```diff
@@ -1026,3 +1026,15 @@ func BackendTLSPolicyMustHaveCondition(t *testing.T, client client.Client, timeo
 
 	require.NoErrorf(t, waitErr, "error waiting for BackendTLSPolicy status to have a Condition %v", condition)
 }
+
+// BackendTLSPolicyMustHaveLatestConditions will fail the test if there are
+// conditions that were not updated
+func BackendTLSPolicyMustHaveLatestConditions(t *testing.T, r *v1alpha3.BackendTLSPolicy) {
```

Comment:
Isn't this a subset of `BackendTLSPolicyMustHaveCondition`?

Reply:
Not exactly, it doesn’t explicitly assert the `observedGeneration` bump, but I agree it’s implicitly covered in the `BackendTLSPolicyMustHaveCondition` since the results of `ConditionsHaveLatestObservedGeneration` is used as a retry condition.